pkg/webui/console/containers/applications-table/index.js

```diff
@@ -51,26 +58,45 @@ const headers = [
 ]
 
 @bind
-export default class ApplicationsTable extends Component {
+class ApplicationsTable extends Component {
 
-  baseDataSelector ({ applications }) {
-    return applications
+  baseDataSelector (state) {
+    return {
+      applications: selectApplications(state),
+      totalCount: selectApplicationsTotalCount(state),
+      fetching: selectApplicationsFetching(state),
+    }
   }
 
   render () {
+    const { error, ...rest } = this.props
+
+    if (error) {
+      throw error
+    }
+
     return (
       <FetchTable
         entity="applications"
         headers={headers}
         addMessage={sharedMessages.addApplication}
-        tableTitle={this.tableTitle}
-        getItemsAction={getApplicationsList}
-        searchItemsAction={searchApplicationsList}
+        getItemsAction={getApplications}
         tabs={tabs}
         baseDataSelector={this.baseDataSelector}
-        {...this.props}
+        {...rest}
       />
     )
   }
 }
 
+ApplicationsTable.propTypes = {
+  error: PropTypes.error,
```

Comment:
Incomplete.

Reply:
would be nice to list what is missing. i documented exactly what i added to the component

Reply:
Sorry, I have no idea why my review got submitted prematurely. I am still in the process of reviewing and added this merely as a reminder for myself. Please disregard this review for the time being until I finished it properly.